src/swift/DispatchStubs.cc

```diff
@@ -153,6 +161,19 @@ static void _dispatch_overlay_constructor() {
   dispatch_sync(queue, block);
 }
 
+#define PUN(t)                                                                \
+SWIFT_CC(swift) DISPATCH_RUNTIME_STDLIB_INTERFACE                             \
+extern "C" dispatch_object_t                                                  \
+_dispatch_pun_##t##_to_object(dispatch_##t##_t obj) {                         \
```

Comment:
Thanks. I'll give it a try.  It would be nice to avoid dropping into C for what really is a no-op.